<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Loader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="box_top">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Stay on Top</string>
    </property>
    <widget class="QRadioButton" name="top_true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>28</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="top_false">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>48</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>No</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box_target">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Target Image</string>
    </property>
    <widget class="QTextBrowser" name="img_url">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pick_img">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>25</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Pick Image</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box_window">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Show Window</string>
    </property>
    <widget class="QRadioButton" name="window_true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>28</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="window_false">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>48</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>No</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box_opacity">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Opacity: 50%</string>
    </property>
    <widget class="QSlider" name="opacity_slider">
     <property name="geometry">
      <rect>
       <x>36</x>
       <y>28</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_low">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>28</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_high">
     <property name="geometry">
      <rect>
       <x>336</x>
       <y>28</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>100%</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="show_img">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Image</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="box_ratio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Maximum Image Ratio: 80%</string>
    </property>
    <widget class="QSlider" name="ratio_slider">
     <property name="geometry">
      <rect>
       <x>36</x>
       <y>28</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>80</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_low_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>28</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_high_2">
     <property name="geometry">
      <rect>
       <x>336</x>
       <y>28</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>100%</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
